analysis/report_templates/default.html

```diff
@@ -16,7 +16,12 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>FuzzBench: {{ experiment.name }} report</title>
+        <title>
+            FuzzBench: {{ experiment.name }} report
+            {% if in_progress %}
+            (running)
```

Comment:
This line is on tab title, so there is no space for that long message. Rest modified as per your suggestion.